app/javascript/stylesheets/bootstrapVariables.scss

```diff
@@ -0,0 +1,2 @@
+// Colors
+$primary: $stanford-digital-blue;
```

Comment:
extracted from main.scss, in alignment with how access team does things.

Note that I'm not sure about the file naming and id naming discrepancy between H2 (camelCase) and e.g. EarthWorks (kebab-case). Happy to take suggestions here for increased consistency and alignment with best practices.